nff/io/ase_calcs.py

```diff
@@ -283,8 +291,9 @@ def offset_energy(self, atoms, energy: Union[float, np.ndarray]):
         for ele, num in ads_count.items():
             ref_en += num * stoidict.get(ele, 0.0)
         ref_en += stoidict.get("offset", 0.0)
+
         if self.offset_units == "atomic":
-            energy += ref_en * HARTREE_TO_EV
+             energy += ref_en * HARTREE_TO_EV
```

Comment:
There seems to be a small indentation mismatch here? Does the code run without errors in this version?